client/styles/components/_console.scss

```diff
@@ -18,28 +18,53 @@
 	}
 
 	// assign styles to different types of console messages
-	.preview-console__log {
-		@include themify(){
-			color: getThemifyVariable('secondary-text-color');
-		}
+	.preview-console__log, .preview-console__info{
 		flex: 1 0 auto;
+		position: relative;
+
+		.preview-console__log__logged-times, .preview-console__info__logged-times{
+			background-color: $lightsteelblue;
+		}
 	}
 
 	.preview-console__undefined {
-		@include themify(){
-			color: getThemifyVariable('inactive-text-color');
-		}
 		flex: 1 0 auto;
+		position: relative;
 	}
 
 	.preview-console__error {
-		color: $console-error-color;
 		flex: 1 0 auto;
+		position: relative;
+
+		.preview-console__error__logged-times {
+			background-color: $red;
+		}
 	}
 
 	.preview-console__warn {
-		color: $console-warn-color;
 		flex: 1 0 auto;
+		position: relative;
+
+		.preview-console__warn__logged-times {
+			background-color: $orange;
+		}
+	}
+
+	.preview-console__debug {
+		flex: 1 0 auto;
+		position: relative;
+
+		.preview-console__debug__logged-times {
+			background-color: $dodgerblue;
+		}
+	}
+
+	.theme-with-dark-background, .theme-with-contrast-background {
```

Comment:
if, as above, you change the class of the console messages to `.preview-console__message--${method}`, then you can change this to
```scss
.preview-console__message {
  @include themify() {
     color: getThemifyVariable('console-bg-color');
  }
}
```

And then you need to add this variable to the `$theme` map in `_variables.scss`.